styleguide/source/_patterns/03-organisms/wayfinder.twig

```diff
@@ -0,0 +1,22 @@
+<section class="ama_wayfinder">
+  <div class="ama_wayfinder--grid container">
+    <div class="ama_wayfinder--referrer">
+      {% if wayfinder.referred %}
+        {% set linkBackUrl = wayfinder.referred.referrerHref %}
+        {% set linkBackTitle = wayfinder.referred.referrerTitle %}
+        <div class="ama_wayfinder--referrer--link-back">
```

Comment:
Since "link back" is really a sub element of `ama__wayfinder--referrer` this should be separated with a double underscore instead of a double hyphen (e.g. `ama__wayfinder--referrer__link-back`).